test/helpers/resvg-preheat.js

```diff
@@ -0,0 +1,7 @@
+// resvg preheat
+const resvg = require('@resvg/resvg-js');
+
+resvg.render('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"></svg>', {
+    fitTo: { mode: 'original' },
```

Comment:
This is the default value, please remove this configuration.

Reply:
Ok thanks